app/Providers/AppServiceProvider.php

```diff
@@ -96,26 +96,26 @@ public function boot()
             return in_array($value, array_keys(Bibliography::bibtexTypes));
         });
         Validator::extend('si_baseunit', function ($attribute, $value, $parameters, $validator) {
-            return in_array($value, array_keys(SiUnitAttribute::getUnits()));
+            return UnitManager::get()->hasQuantity($value);
```

Comment:
Moved that functionality to the 'UnitManager'. I'm unsure about the naming. In English it's called the 'quality' in which 'domain' the value resided e.g. length or area. But otherwise it's just the name of the UnitSystem in our context. So that's debatable.